src/ui/public/vis/agg_config.js

```diff
@@ -298,10 +299,22 @@ export function VisAggConfigProvider(Private) {
     }
 
     if (!this.type) return '';
-    let pre = (_.get(this.vis, 'params.mode') === 'percentage') ? 'Percentage of ' : '';
+    let pre = (this.isPercentage()) ? 'Percentage of ' : '';
     return pre += this.type.makeLabel(this);
   };
 
+  AggConfig.prototype.isPercentage = function () {
```

Comment:
actually this is creating couplings that shouldn''t be there ... agg_config shouldn't have to know anything about vislib.
also it probably shouldn't have to know anything about percentage options and such.

i think vislib should take care of this when rendering the tooltip.